data/meson.build

```diff
@@ -82,7 +87,7 @@ if build_daemon
     endif
     if systemd.version().version_compare('>= 231')
       dynamic_options += 'RestrictRealtime=yes'
-#     dynamic_options += 'MemoryDenyWriteExecute=yes'
+      #     dynamic_options += 'MemoryDenyWriteExecute=yes'
```

Comment:
I think this should be `#dynamic_options`

Reply:
Agree, fixed.